src/coreclr/jit/targetx86.h

```diff
@@ -74,6 +74,10 @@
 
   #define REG_FP_FIRST             REG_XMM0
   #define REG_FP_LAST              REG_XMM7
+
+  #define REG_MASK_FIRST           REG_K0
+  #define REG_MASK_LAST            REG_K7
```

Comment:
Since you will be just using `REG_K1`, why not just enable single register and do the rest in follow-up PR when you consume more of those?

Reply:
I can make the change.

Is there a reason not to just enable all of them in this pass though? Is it for testing purposes?

Reply:
Presumably having it be more correct here when the initial support is being added will make it less error prone and overall simpler when its extended to be more than just a single selection, right?

Reply:
That sounds fine to me. I just wanted to understand the reasoning behind adding all the registers even though they are not consumed.